src/NServiceBus.RabbitMQ/DelayedDelivery/DelayInfrastructure.cs

```diff
@@ -16,7 +16,10 @@ static class DelayInfrastructure
 
         public const int MaxDelayInSeconds = (1 << maxNumberOfBitsToUse) - 1;
         public const string DelayHeader = "NServiceBus.Transport.RabbitMQ.DelayInSeconds";
-        public const string DeadLetteredMessageHeader = "x-death";
+        public const string DeadLetteredMessageHeader1 = "x-death";
+        public const string DeadLetteredMessageHeader2 = "x-first-death-exchange";
+        public const string DeadLetteredMessageHeader3 = "x-first-death-queue";
+        public const string DeadLetteredMessageHeader4 = "x-first-death-reason";
```

Comment:
I'm not really happy with these names, but I wasn't sure what to use instead. Maybe go with `DeathHeader`, `FirstDeathExchangeHeader`, etc?

Reply:
How about we use the same names, and  just convert the `kekab-case` to `PascalCase`, with the `Header` suffix? I.e. `XDeathHeader`, etc.